src/actions/teamMemberActions.js

```diff
@@ -0,0 +1,17 @@
+import { LOAD_TEAM_MEMBERS_SUCCESS } from './actionTypes';
```

Comment:
the good practise is to dispatch `LOAD_TEAM_MEMBERS_REQUEST` before the request and `LOAD_TEAM_MEMBERS_FAILURE` after failure :)
Even if you don't do anything with it at this point. It will make debugging with redux devtools easier.